Add security information (#26)
0xAX/asm: SECURITY.md

# Security information

> The [repository](https://github.com/0xAX/asm) contains code examples written mostly using [C](https://en.wikipedia.org/wiki/C_(programming_language)) and [assembly](https://en.wikipedia.org/wiki/Assembly_language) programming languages. All of the code examples do not contain any malicious code and could be safely launched on your local computer.

Reviewer: ```suggestion
This repository contains code examples written mostly using [C](https://en.wikipedia.org/wiki/C_(programming_language)) and [assembly](https://en.wikipedia.org/wiki/Assembly_language) programming languages. The code examples do not contain any malicious software and can be safely run on your local machine.
```

Author: :+1:

> In any cases if you have reason to believe that there are security issues across any code examples, please feel free to open the [issue](https://github.com/0xAX/asm/issues/new).

Reviewer: ```suggestion
If you have reason to believe there are security issues across any code examples, please open an [issue](https://github.com/0xAX/asm/issues/new).
```
